master/getting-started/mesos/installation/dc-os/framework.md

```diff
@@ -57,128 +35,35 @@ You should see a page that looks like the following:
 
 ![alt tag]({{site.baseurl}}/images/mesos/dcos-calico-status.png)
 
->Note: The webpage may fail to load initially since the Calico
-service restarts the Docker and Mesos agent services, which
-will cause a restart of the Calico framework when it happens
-on the same agent. This is expected and nothing to be
-concerned about. Once the status page is running, you will see a
-table where each row represents an agent, including statuses
-for each of the Calico components installed.
+## Further Reading
+
+This concludes the installation of Calico for DC/OS! Before you start
+launching IP-per-container applications with Calico policy,
+review the following information which may apply to your deployment.
+
+#### AWS
+
+DC/OS users on Amazon Web Services should view
+[Calico's AWS reference]({{site.baseurl}}/{{page.version}}/reference/public-cloud/aws)
+and be sure to enable IP-IP if their cluster is deployed across multiple
```

Comment:
"and be sure to enable IP-IP if their..." should be "for information on how to configure AWS networking for use with Calico"